crates/top/rerun_c/src/video.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{CError, CErrorCode, CStringView};
+
+#[allow(unsafe_code)]
+#[no_mangle]
+pub extern "C" fn rr_video_asset_read_frame_timestamps_ns(
+    video_bytes: *const u8,
+    video_bytes_len: u64,
+    media_type: CStringView,
+    alloc_context: *mut std::ffi::c_void,
+    alloc_func: Option<
+        extern "C" fn(context: *mut std::ffi::c_void, num_timestamps: u32) -> *mut i64,
+    >,
+    error: *mut CError,
+) -> *mut i64 {
```

Comment:
This bare pointer in the signature seems kind of awkward and hard-to-use without knowledge of the `num_timestamps` param that was passed into the alloc_func.

Do we need this for anything / does it make some common pattern more ergonomic?

Reply:
needed it orginally but didn't bother to remove it. Let's clean this up!